hw/dv/data/xcelium/xcelium.hjson

```diff
@@ -1,6 +1,8 @@
 // Copyright lowRISC contributors.
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
 // SPDX-License-Identifier: Apache-2.0
+//  build_cmd:  "nc run -C ncsim_interactive -Il xrun"
+//  run_cmd:    "nc run -C ncsim_batch -N default -w xrun"
```

Comment:
These changes are not needed for this PR, are they?

Reply:
shoot thanks phil.. nope they are for something very different..

Reply:
@imphil thank you for the feedback both should be fixed now